src/send/types.ts

```diff
@@ -383,33 +384,24 @@ export interface RoutingStrategyProvider<T = Record<string, any>> {
   if?: string;
 }
 
-export interface BaseMessageMetadata {
-  // tags may be an array of up to nine strings (30 characters in length)
-  tags?: [
-    string?,
-    string?,
-    string?,
-    string?,
-    string?,
-    string?,
-    string?,
-    string?,
-    string?
-  ];
-}
 
-export interface ContentMessageMetadata extends BaseMessageMetadata {
+export interface MessageMetadata {
   event?: string;
+  tags?: string[];
+  utm?: {
+    source?: string;
+    medium?: string;
+    campaign?: string;
+    term?: string;
+    content?: string;
+  };
 }
 
 export interface ContentMessage extends BaseMessage {
   content: Content;
-  metadata?: ContentMessageMetadata;
 }
 
 export interface TemplateMessage extends BaseMessage {
-  brand?: string;
```

Comment:
Not blocking. But we should add `brand_id` to the next release. It is supported by the backend.